src/Symfony/Component/Security/Core/Authorization/Voter/Vote.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Core\Authorization\Voter;
+
+class Vote
```

Comment:
A Vote is something returned by a Voter written by the dev in user land. I think it could be useful to extend a Vote for adding extra domain features. But at first, we can make it final, and wait and see if someone needs it.